Userland/Libraries/LibAudio/ConnectionToServer.cpp

```diff
@@ -27,8 +27,10 @@ ConnectionToServer::ConnectionToServer(NonnullOwnPtr<Core::Stream::LocalSocket>
         Core::EventLoop enqueuer_loop;
         m_enqueuer_loop = &enqueuer_loop;
         enqueuer_loop.exec();
-        Threading::MutexLocker const locker(m_enqueuer_loop_destruction);
-        m_enqueuer_loop = nullptr;
+        {
```

Comment:
Isn't this change equivalent to what was here before?

Reply:
Nope; The return should not hold the lock anymore. Either way, this is more explicit about the scope of the lock.